lib/aur-chroot

```diff
@@ -1,130 +0,0 @@
-#!/bin/bash
-# aur-chroot - build packages with systemd-nspawn
```

Comment:
If we are removing the command, and its autocompletion, we should also delete its man page. Parts of it can be added as a `CHROOT` section in `man1/aur-build.1`, or maybe as a `.7` man page.

Reply:
Agreed - I'm working on `aurchroot.7`, see https://github.com/AladW/aurutils/commit/989a80015e22d68ed999a6e97be54e981845217b